src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -273,7 +298,7 @@ String serializeGcpCloudContext(GcpCloudContext gcpCloudContext) {
   }
 
   @VisibleForTesting
-  GcpCloudContext deserializeGcpCloudContext(String json) {
+  static GcpCloudContext deserializeGcpCloudContext(String json) {
```

Comment:
Since the Dao is a singleton bean, is the static needed?

Reply:
yes, but only so I can access this from the static RowMapper above. I could also change `serializeGcpCloudContext` for consistency, or just leave that as-is for now.